dev-libs/sway/sway-0.11-r1.ebuild

```diff
@@ -53,6 +53,7 @@ src_configure() {
 		-Dzsh-completions=$(usex zsh-completion)
```

Comment:
please change `default` in `src_prepare` to `cmake-utils_src_prepare`, as `default` should not be used explicitly with cmake